tests/lib/rules/spaced-comment.js

```diff
@@ -270,8 +273,9 @@ ruleTester.run("spaced-comment", rule, {
         },
         {
             code: "//*********************-\n// Comment Block 3\n//***********************",
+            output: "//* ********************-\n// Comment Block 3\n//***********************",
```

Comment:
You have missed the minus at the end of the first line: `//*********************-\n` that's why the first line in that comment was treated as a error and not as a valid exception in the first place, whoever added that test did it exactly to for that minus. And that's why my code decided to add a space in the first line, but not in the third line where it is indeed a valid exception.